settings.gradle.kts

```diff
@@ -1 +1,5 @@
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.0"
```

Comment:
there seems to have a newer version [0.9.0](https://github.com/gradle/foojay-toolchains/releases/tag/foojay-toolchains-plugin-0.9.0) of the plugin. Since you introduce it in this pr maybe we could directly use the newest version if its compatible :wink: